configure.ac

```diff
@@ -5371,7 +5371,9 @@ PAC_GET_EXENAME("mpifort", MPIFORT_NAME)
 PAC_GET_EXENAME("mpif90", MPIF90_NAME)
 PAC_GET_EXENAME("mpif77", MPIF77_NAME)
 
+AC_CONFIG_SUBDIRS([test/mpi/dtpools])
```

Comment:
@pavanbalaji a possible workaround is to build `dtpools` from main mpich and install it as external package. Then using it in the test suite is just a matter of adding the right `CFLAGS` and `LDFLAGS`. This is just an attempt but I have the suspect you won't like it :)